src/parameter_indexing.jl

```diff
@@ -15,6 +15,56 @@ parameter_values(arr::AbstractArray) = arr
 parameter_values(arr::AbstractArray, i) = arr[i]
 parameter_values(prob, i) = parameter_values(parameter_values(prob), i)
 
+"""
+    parameter_values_at_time(p, t)
+
+Return an indexable collection containing the value of all parameters in `p` at time index
+`t`. This is useful when parameter values change during the simulation
+(such as through callbacks) and their values are saved. `t` is the time index in the
+timeseries formed by these changing parameter values, obtained using
+[`parameter_timeseries`](@ref).
+
+By default, this function returns `parameter_values(p)` regardless of `t`, and only needs
+to be specialized for timeseries objects where parameter values are not constant at all
+times. The resultant object should be indexable using [`parameter_values`](@ref).
+
+If this function is implemented, [`parameter_values_at_state_time`](@ref) must be 
+implemented for [`getu`](@ref) to work correctly.
+"""
+function parameter_values_at_time end
+parameter_values_at_time(p, t) = parameter_values(p)
+
+"""
+    parameter_values_at_state_time(p, t)
+
+Return an indexable collection containing the value of all parameters in `p` at time
+index `t`. This is useful when parameter values change during the simulation (such as
```

Comment:
same here

Reply:
Fixed in https://github.com/SciML/SymbolicIndexingInterface.jl/pull/53/commits/cc5da9083caa15e894a4ea2f8d35300b0fb27ff4